metaStoreApp/spec/models/video_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe Video, type: :model do
+  it "should update timecode based on the value of duration passed" do
+    @account = FactoryBot.create(:account)
+    @metadata = FactoryBot.build(:video, :account_id => @account.id, :duration => 12000)
```

Comment:
why to user build over create?

Reply:
build will not save the object into DB but will trigger validations only for associated objects while create call save method, thus build is faster than create